src/proxy/proxypython.cpp

```diff
@@ -197,9 +197,14 @@ QString ProxyPython::name() const
   return "Python Proxy";
 }
 
+QString ProxyPython::localizedName() const
+{
+  return tr("Python Proxy");
+}
+
 QString ProxyPython::author() const
 {
-  return "Tannin";
+  return "Tannin & Holt59";

```

Comment:
If you're bothering with this, then I think me and Silarn need adding, too.

Reply:
I don't usually bother but since I've refactored most of the code during the last few months... Anyway, I've added both names.